app/models/json_web_token.rb

```diff
@@ -10,6 +10,6 @@ def self.encode(payload, exp = 24.hours.from_now)
 
   def self.decode(token)
     decoded = JWT.decode(token, SECRET_KEY)[0]
-    HashWithIndifferentAccess.new decoded
+    ActiveSupport::HashWithIndifferentAccess.new decoded
```

Comment:
Isso aqui seria a mesma coisa que isso?
```suggestion
    decoded.with_indifferent_access
```